ndsl/dsl/dace/orchestration.py

```diff
@@ -99,34 +88,39 @@ def _to_gpu(sdfg: dace.SDFG):
         else:
             arr.storage = dace.StorageType.GPU_Global
 
-    # All maps will be scedule on GPU
-    for mapentry, state in topmaps:
+    # All maps will be schedule on GPU
+    for mapentry, _state in topmaps:
         mapentry.schedule = dace.ScheduleType.GPU_Device
 
     # Deactivate OpenMP sections
     for sd in sdfg.all_sdfgs_recursive():
         sd.openmp_sections = False
 
 
-def _simplify(sdfg: dace.SDFG, validate=True, verbose=False):
+def _simplify(
+    sdfg: dace.SDFG,
+    *,
+    validate: bool = True,
+    validate_all: bool = False,
+    verbose: bool = False,
+):
     """Override of sdfg.simplify to skip failing transformation
     per https://github.com/spcl/dace/issues/1328
     """
     return SimplifyPass(
         validate=validate,
+        validate_all=validate_all,
         verbose=verbose,
         skip=["ConstantPropagation"],
     ).apply_pass(sdfg, {})
 
 
 def _build_sdfg(
-    daceprog: DaceProgram, sdfg: dace.SDFG, config: DaceConfig, args, kwargs
+    program: DaceProgram, sdfg: dace.SDFG, config: DaceConfig, args, kwargs
```

Comment:
We should keep `dace_program` because `gt4py.next` comes with it's own concept of `program`. Bit of defensive programming there

Reply:
Makes total sense and I'm happy to rename this to `dace_program`